torchgeo/datamodules/vaihingen.py

```diff
@@ -3,13 +3,19 @@
 
 """Vaihingen datamodule."""
 
-from typing import Any, Dict, Optional
+from typing import Any, Dict, List, Optional, Tuple, Union
 
+import kornia.augmentation as K
 import matplotlib.pyplot as plt
 import pytorch_lightning as pl
+import torch
+from einops import rearrange
 from torch.utils.data import DataLoader, Dataset
+from torch.utils.data._utils.collate import default_collate
 from torchvision.transforms import Compose
 
+from torchgeo.samplers.utils import _to_tuple
```

Comment:
```suggestion
from ..samplers.utils import _to_tuple
```
I prefer to use relative imports within torchgeo